pkg/httpd/simpleserver.go

```diff
@@ -48,12 +49,26 @@ func (simpleServer *SimpleServer) Start() error {
 	mux.HandleFunc("/healthz", webhook.HealthCheckHandler)
 	mux.HandleFunc("/mutate", admissionHandler.HandleAdmission)
 
+	addr := ":9090"
+	startMetricsServer(addr)
```

Comment:
Pretty sure this needs to be done in a go routine. otherwise it will not make it to actually run the other server.

Reply:
updated